hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/QueuePath.java

```diff
@@ -105,6 +108,14 @@ public boolean hasEmptyPart() {
     return false;
   }
```

Comment:
Have you added UTs for all new functionality of this class?

Reply:
Added the missing unit tests for the new functionalities